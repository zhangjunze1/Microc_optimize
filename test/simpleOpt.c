void main() {
    int i=1;
    i++;
    print("%d",i);
    i--;
    print("%d",i);
    i+=2;
    print("%d",i);
    i-=2;
    print("%d",i);
    i*=2;
    print("%d",i);
    i/=2;
    print("%d",i);
    i%=2;
    print("%d",i);
}
