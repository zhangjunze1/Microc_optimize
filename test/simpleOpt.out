24 19 0 5 25 13 0 1 12 0 1 0 1 1 15 -1 13 13 11 0 1 1 12 15 -1 13 11 22 15 -1 0 1 0 1 2 15 -1 13 13 11 0 1 2 12 15 -1 13 11 22 15 -1 0 2 13 13 11 0 2 1 12 15 -2 13 11 22 15 -1 0 2 13 13 11 0 2 2 12 15 -2 13 11 22 15 -1 0 2 13 13 11 0 2 3 12 15 -2 13 11 22 15 -1 0 2 13 13 11 0 2 4 12 15 -2 13 11 22 15 -1 0 2 13 13 11 0 2 5 12 15 -2 13 11 22 21 1